lib/types/api.types.ts

```diff
@@ -274,6 +316,10 @@ export interface ConnectMessage {
   ids: IdsAPI
   contentType: ContentTypeAPI
   editorInterface?: EditorInterface
+  editor?: {
+    localeSettings: EditorLocaleSettings
+    showDisabledFields: boolean
```

Comment:
@kdamball Probably we should also change the prop name of the connect message, right? That way the connect message would have a different name than the event (`showDisabledFieldsChanged`).

Reply:
I would change everything to match this new property name and remove any doubt